package.json

```diff
@@ -47,6 +47,7 @@
     "react-native-screens": "^2.2.0",
     "react-native-svg": "9.13.3",
     "react-native-swipe-list-view": "^3.0.0",
+    "react-native-text-ticker": "^1.3.0",
```

Comment:
It doesn't look like this is being used unless I missed something

Reply:
Yeah it's not I can remove it